packages/wdio-jasmine-framework/jasmine-framework.d.ts

```diff
@@ -13,6 +15,54 @@ declare namespace WebdriverIO {
     }
 }
 
-declare module "@wdio/jasmine-framework" {
-    export default WebdriverIO
+interface JasmineNodeOptsConfig {
+    jasmineNodeOpts?: JasmineNodeOpts
+}
+
+interface JasmineNodeOpts {
```

Comment:
Can you please add `requires` to Jasmine node opts?
https://github.com/webdriverio/webdriverio/blob/993fa84fac2c28e15753ed02bd249bbfcdac0135/packages/wdio-jasmine-framework/src/index.js#L152

It might be a duplicated comment, sorry if so

Reply:
this is not supported by Jasmine anymore. I will remove this.